doc/configuration-v3_0-experimental.md

```diff
@@ -104,7 +104,7 @@ The Ignition configuration is a JSON document conforming to the following specif
   * **_users_** (list of objects): the list of accounts that shall exist.
     * **name** (string): the username for the account.
     * **_passwordHash_** (string): the encrypted password for the account.
-    * **_sshAuthorizedKeys_** (list of strings): a list of SSH keys to be added to the user's authorized_keys.
+    * **_sshAuthorizedKeys_** (list of strings): a list of SSH keys to be added as an SSH key fragment at `authorized_keys.d/ignition` in the user's .ssh directory. To properly read the keys added by Ignition and allow the user to log in, the distribution should support reading key fragments from `authorized_keys.d`. For distributions that do not support `authorized_keys.d`, the keyfile written by Ignition may be configured as `authorized_keys` through the Go linker flag `-X github.com/coreos/ignition/internal/distro.useAuthorizedKeysFile=true`.
```

Comment:
"`.ssh/authorized_keys.d/ignition` in the user's home directory" would be clearer.

I don't think distro integration should be mentioned in the spec.  It should go in the docs requested in #649.

Reply:
Thanks - makes sense! Will have a think on the docs there.